cli/command/service/opts.go

```diff
@@ -649,9 +619,6 @@ const (
 	flagPublish               = "publish"
 	flagPublishRemove         = "publish-rm"
 	flagPublishAdd            = "publish-add"
-	flagPort                  = "port"
-	flagPortAdd               = "port-add"
-	flagPortRemove            = "port-rm"
```

Comment:
I don't think we can just remove `--port` they were in the 1.12 release as non-experimental. We have to deprecate them.